tests/testmodels.py

```diff
@@ -257,10 +258,16 @@ class JSONFields(Model):
     This model contains many JSON blobs
     """
 
+    @staticmethod
+    def dict_or_list(value: Union[dict, list]):
+        if not isinstance(value, (dict, list)):
+            raise ValidationError("Value must be a dict or list.")
+
     id = fields.IntField(pk=True)
     data = fields.JSONField()
     data_null = fields.JSONField(null=True)
     data_default = fields.JSONField(default={"a": 1})
+    data_validate = fields.JSONField(null=True, validators=[lambda v: JSONFields.dict_or_list(v)])
```

Comment:
Putting the field on this model means the 'describe' tests need to be updated, but it feels more appropriate than putting it on the validation model as this is testing the `JSONField` itself, not packaged validations. I've used a static method on the object rather than a module method as the validator to keep things self-contained, but happy to change if you find it too clumsy.